cloudflare/resource_cloudflare_worker_kv.go

```diff
@@ -0,0 +1,127 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/pkg/errors"
+)
+
+func resourceCloudflareWorkerKV() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareWorkersKVCreate,
+		Read:   resourceCloudflareWorkersKVRead,
+		Update: resourceCloudflareWorkersKVUpdate,
+		Delete: resourceCloudflareWorkersKVDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareWorkersKVImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"key": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"namespace_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"value": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceCloudflareWorkersKVRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*cloudflare.API)
+	namespaceID, key := parseId(d)
+
+	value, err := client.ReadWorkersKV(context.Background(), namespaceID, key)
```

Comment:
All of the Worker KV methods require setting [`api.AccountID`](https://github.com/cloudflare/cloudflare-go/blob/4a4622ed71d4de360d64a887e3cc3417d41ed299/workers_kv.go#L203) however we're not setting it or documenting it as a required configuration parameter. How does it work at the moment?

Reply:
None of the methods have a parameter for the account ID, it is set on the cloudflare.API client when the provider configures it.  This is identical to how the existing cloudflare_workers_kv_namespace resource works, but I can add it to the documentation if needed.

Reply:
Added a note in the docs that setting `account_id` on the provider, or the `CLOUDFLARE_ACCOUNT_ID` environment variable is necessary to use this.  Does that cover the requirement?